tests/unit/resource/CMakeLists.txt

```diff
@@ -37,6 +41,7 @@ foreach(test ${tests})
     ${${test}_FLAGS}
     EXCLUDE_FROM_ALL
     HPX_PREFIX ${HPX_BUILD_PREFIX}
+    DEPENDENCIES hpx_debugging hpx_testing
```

Comment:
No need for these. `add_hpx_executable` links to libhpx (if it doesn't we fix that instead).